hera_cal/frf.py

```diff
@@ -1361,7 +1361,8 @@ def tophat_frfilter(self, frate_centers, frate_half_widths, keys=None, wgts=None
 def time_avg_data_and_write(input_data_list, output_data, t_avg, baseline_list=None,
                             wgt_by_nsample=True, wgt_by_favg_nsample=False, rephase=False,
                             filetype='uvh5', verbose=False, clobber=False, flag_output=None,
-                            ninterleave=1, **read_kwargs):
+                            ninterleave=1, equalize_interleave_times=True,
```

Comment:
These are kinda weird, destructive choices reflecting non-idealities of hera_pspec. Seems to me like default False makes more sense...

Reply:
Done.